File: j2k/tests/testData/fileOrElement/tryWithResource/WithCatches.java
//file
import java.io.*;

public class C {
    void foo() {
        try(InputStream stream = new FileInputStream("foo")) {
            // reading something
            int c = stream.read();
            System.out.println(c);
        }
        catch (IOException e) {
            System.out.println(e);
        }
        catch (Exception e) {
            System.err.println(e);
        }
    }
}

File: j2k/tests/testData/fileOrElement/tryWithResource/WithCatches.kt
import java.io.*

public class C {
    fun foo() {
        try {
            ByteArrayInputStream(ByteArray(10)).use { stream ->
                // reading something
                val c = stream.read()
                System.out.println(c)
            }
        } catch (e: IOException) {
            System.out.println(e)
        } catch (e: Exception) {
            System.err.println(e)
        }

    }
}